kuma/javascript/src/payments/subheader.jsx

```diff
@@ -0,0 +1,35 @@
+//@flow
+import * as React from 'react';
+
+import { gettext } from '../l10n.js';
+
+type Props = {
+    title: string,
+    subtitle?: string,
+    description?: string,
+    children?: React.Node,
+    columnWidth?: string
```

Comment:
Is it really optional?

Reply:
Yeah, because we set a default value for `columnWidth` ("all") below

Reply:
bah! I meant the line above. `children` Sorry. Really not sure it matters at all.

Reply:
I see-- when I remove `children`, it complains that that it wasn't in the Props declaration, so we need it here.

Reply:
It does-- I got a complaint when I removed the `?` saying that children was required in `<SubHeader />` but not provided